### PR TITLE
Fix quickstart for 4.1.x

### PR DESCRIPTION
**Pull request by joel-hamill**
Removes legacy .md files and docker instructions (Docker is not available for 4.1 GA).

### CODE REVIEW

File: docs/quickstart/index.rst [hunk 1]
```diff
@@ -3,39 +3,149 @@
...
+
+Run these steps to produce data to the Kafka topics ``pageviews`` and ``users``.
+
+1. Produce Kafka data to the ``pageviews`` topic using the data
+   generator. The following example continuously generates data with a
+   value in DELIMITED format.
+
+   .. code:: bash
+
+       $ ./bin/ksql-datagen quickstart=pageviews format=delimited topic=pageviews maxInterval=10000
```
Comment (hjafarpour) [inline docs/quickstart/index.rst]: Let's change this to `maxInterval=500`

File: docs/quickstart/index.rst [hunk 2]
```diff
@@ -3,39 +3,149 @@
...
+
+2. Produce Kafka data to the ``users`` topic using the data generator.
+   The following example continuously generates data with a value in
+   JSON format.
+
+   .. code:: bash
+
+       $ ./bin/ksql-datagen quickstart=users format=json topic=users maxInterval=10000
```
Comment (hjafarpour) [inline docs/quickstart/index.rst]: Also let's change this to `maxInterval=1000`

File: docs/quickstart/index.rst (hunk 1 above)
Comment (dguy) [reply docs/quickstart/index.rst]: @hjafarpour any reason whey we don't make this even smaller? i.e, say 100?

**Comment by hjafarpour** [comment]
@dguy no reason, thought table can be updated with less frequency than the stream. We can make it 100.

**Pull request merged**